src/main/resources/openapi/usermanagement.yaml

```diff
@@ -168,6 +168,36 @@ paths:
                 title: "The provided security group does not exist"
                 status: 404
 
+    patch:
+      summary: Patch security group
+      tags:
+        - SecurityGroup
+      operationId: modifySecurityGroup
+      parameters:
+        - $ref: '#/components/parameters/SecurityGroupId'
+      requestBody:
+        $ref: '#/components/requestBodies/SecurityGroupPatchRequest'
+      responses:
+        '200':
+          $ref: '#/components/responses/SecurityGroupPatchSuccess'
+        '403':
```

Comment:
I'd expect to see the other client errors listed here also - e.g. 404 you have `UserManagementError.SECURITY_GROUP_NOT_FOUND` and `UserManagementError.COURTHOUSE_NOT_FOUND` and
`UserManagementError.USER_NOT_FOUND`?

There is also 400, 401 errors?

Also 409? `DUPLICATE_SECURITY_GROUP_NAME_NOT_PERMITTED`

If you can check them please